### DISCUSSION TITLE
MSA Tokenization

### DISCUSSION THREAD

**Discussion by sbpkyle** [chaidiscovery/chai-lab#71] [Q&A] [answered]
Hi, I was wondering if there was a method in place to tokenize a MSA file. I tried looking and there wasn't obvious to me that was doing this. 

Thanks!

**Comment by jackdent** [comment, marked as answer]
~Hi @sbpkyle, there isn't yet, but we're planning to release some very soon. Stay tuned. cc @arogozhnikov~

MSAs are now supported: https://github.com/chaidiscovery/chai-lab/tree/main/examples/msas

**Reply by sbpkyle** [reply]
> Hi @sbpkyle, there isn't yet, but we're planning to release some very soon. Stay tuned. cc @arogozhnikov

Hi @jackdent and @arogozhnikov, I happen to notice you released a new version and saw some of the changes in which the changes are related to the MSA. Is it working now or are you all still working on it?

**Reply by arogozhnikov** [reply]
it is ready to use! we added a section to repo readme how to get started
